app/views/foreman_ansible/hostgroups/_tab_content.html.erb

```diff
@@ -0,0 +1,3 @@
+<div class='tab-pane' id='ansible_roles'>
+  <%= render 'foreman_ansible/ansible_roles/select', :object => @hostgroup, :f => f %>
```

Comment:
No need to pass any `object` given https://github.com/theforeman/foreman_ansible/pull/30/files#r74913763